src/main.py

```diff
@@ -0,0 +1,130 @@
+from typing import Optional
+
+import torch.optim
+import pandas as pd
+import torch.nn as nn
+from torch.utils.data import DataLoader
+
+from model import PitchSeqNN
+from src.data.dataset import ComposerClassificationDataset
+
+BATCH_SIZE = 16
+N_EPOCHS = 100
+
+
+def prepare_dataset_pitch_only(dataset: ComposerClassificationDataset):
+    """
+    Prepares the dataset by selecting only the pitch information from the samples and converting it
+    into a format suitable for training.
+
+    Args:
+        dataset (ComposerClassificationDataset): An instance of YourDatasetClass containing samples with 'notes'
+                                    and 'composer' information.
+
+    Returns:
+        None: Modifies the 'samples' attribute of the dataset in-place.
+    """
+
+    def divide(lst):
+        return [num / 127.0 for num in lst]
+
+    df = pd.DataFrame(dataset.samples)
+    df = df[["notes", "composer"]]
+    # change classnames to numbers
+    df["composer"] = df["composer"].apply(lambda x: 0 if x == dataset.selected_composers[0] else 1)
+    # take only ['pitch'] column as an input tensor, normalize it to belong to (0, 1]
+    df["notes"] = df["notes"].apply(lambda x: x["pitch"])
+    df["notes"] = df["notes"].apply(divide)
```

Comment:
```suggestion
    df["pitches"] = df["notes"].apply(lambda x: x["pitch"])
    df["normalized_pitches"] = df["notes"].apply(divide)
```

I think overwriting existing columns can quickly lead to confusion. I would also suggest looking for a different way to do the normalization, maybe even after the tensor is created from the dataframe, rather than inside of it.